### PR TITLE
Calls: Add content on datachannels and turn

### PR DESCRIPTION
**Pull request by renandincer**

### CODE REVIEW

File: content/calls/datachannels.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+pcx_content_type: get-started
+title: DataChannels
+weight: 8
+---
+
+# DataChannels
+
+Since Cloudflare Calls is basically a pub/sub server for WebRTC that can scale up to many subscribers per publisher, it's fit for arbitrary data besides media too.
```
Comment (rcccf) [inline content/calls/datachannels.md:9]: Could we also make it explicit that initial implementation:

1. only supports `ordered` and `reliable` mode (sort of like websocket but over webrtc);
2. data sent back to a "subscribed" channel (aka a pulled channel) will be ignored

**Review by third774 — APPROVED** [review]
LGTM, but we should update to use "allowlist"

**Comment by third774** [comment]
> LGTM, but we should update to use "allowlist"

Oh, wait — nevermind, looks like we're using both here?

**Comment by renandincer** [comment]
Rachel suggestion will come in next update.

@third774: yeah I was trying to use both in at least 1 place for searchability

**Pull request merged**